pint/models/parameter.py

```diff
@@ -144,7 +144,7 @@ def base_value(self):
     def base_value(self,val):
         self._base_value = val
         if not isinstance(val, numbers.Number):
-            return
```

Comment:
If we are not deal with complex numbers, I believe numbers.Real works as
well.
-Luo Jing

On Tuesday, January 26, 2016, Scott Ransom <notifications@github.com> wrote:

> In pint/models/parameter.py
> <https://github.com/nanograv/PINT/pull/86#discussion_r50936870>:
>
> > @@ -144,7 +144,7 @@ def base_value(self):
> >      def base_value(self,val):
> >          self._base_value = val
> >          if not isinstance(val, numbers.Number):
> > -            return
>
> Might it not be better to compare vs numbers.Real rather than
> numbers.Number? What are the advantages of numbers.Number?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nanograv/PINT/pull/86/files#r50936870>.
>